geoh5py/io/h5_writer.py

```diff
@@ -559,6 +559,10 @@ def write_data_values(
                     compression="gzip",
                     compression_opts=9,
                 )
+                entity_type_handle = H5Writer.fetch_handle(h5file, entity.entity_type)
```

Comment:
**question:** is it per entity type? I was expecting it to be per "data type"

@jincandescent , do you confirm?

Reply:
here the entity is a "data" entity.

Reply:
mmh... however the data type is not the `entity_type`, or is it?

Reply:
data type is how the user regroups data for statistics. Typically done automatically based  on the name of the data.
@jincandescent  correct me if I am wrong

Reply:
Under geoh5py, the data_type is just a sub-class of entity_type, without a fix uuid
![image](https://user-images.githubusercontent.com/55204635/142667750-ba296049-5706-4686-8e65-f9c894ac296c.png)